api/spec/spec_helper.rb

```diff
@@ -15,6 +15,8 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 require "fakeweb"
+# Library for stubbing and setting expectations on HTTP requests in Ruby.
+require 'webmock/rspec'
```

Comment:
Kind of odd, but without fakeweb, he api was trying to do the real calls.
With it, there was an error explicitly describing which api was not stubbed, which was very handy.